prow/plugins/label/label.go

```diff
@@ -226,19 +230,28 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	}
 
 	toRepeat := []string{}
+	isMember := false
 	if len(sigMatches) > 0 {
-		isMember, _ := gc.IsMember(ae.org, ae.login)
-		if !isMember {
-			toRepeat = ae.getRepeats(sigMatches, existingLabels)
-		}
+		isMember, _ = gc.IsMember(ae.org, ae.login)
+		toRepeat = ae.getRepeats(sigMatches, existingLabels)
 	}
 
 	if len(toRepeat) > 0 {
 		msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
-		if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
-			log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
+		if !isMember {
+			if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
+				log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
+			}
 		}
 
+		if sc != nil {
+			//if sig matches then send a notification on slack
+			for _, sig := range sigMatches {
+				if err := sc.WriteMessage(plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg), "sig-"+sig[1]); err != nil {
```

Comment:
Yeah just the comment body isn't enough. Including the issue/PR title and number is probably enough to provide adequate context. You'll definitely also want to include the ae.url somewhere in the message so that people who view the slack message can easily find the github comment that triggered the message.